src/components/Router/Flow.js

```diff
@@ -0,0 +1,91 @@
+import { h, Component } from 'preact'
+import createHistory from 'history/createBrowserHistory'
+
+import { events } from '../../core'
+import {sendScreen} from '../../Tracker'
+import {wrapArray} from '../utils/array'
+
+const history = createHistory()
+
+class Flow extends Component {
```

Comment:
I don't like this component name - I would prefer StepsRouter or LinearRouter.

I also think the names of the 'flows' i.e. 'master' and 'crossDevice' could be made clearer. crossDevice is OK, but I think master could be called 'upload' or something more descriptive

Reply:
or `zeus` since he is the master of gods

Reply:
joking...

Reply:
Yes, I agree. `StepsRouter` makes sense instead of `Flow` and I will replace 'master' with 'captureSteps' and 'crossDevice' with 'crossDeviceSteps'